thoth/storages/data/alembic/versions/5c1e7425ce60_added_entry_in_.py

```diff
@@ -0,0 +1,32 @@
+"""Added entry in KebechetGithubInstallation table for environment name.
+
+Revision ID: 5c1e7425ce60
+Revises: 65ae36e5c38a
+Create Date: 2021-08-10 15:38:46.589634+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '5c1e7425ce60'
+down_revision = '65ae36e5c38a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('kebechet_github_installations', sa.Column('runtime_environment_name', sa.Text(), nullable=True))
+    op.create_unique_constraint(None, 'kebechet_github_installations', ['slug', 'runtime_environment_name'])
+    op.create_unique_constraint(None, 'python_package_version_entity_rule', ['id'])
```

Comment:
This does not seem to be part of this migration right? Also in Downgrade

Reply:
Removed

Reply:
there is also `python_package_version_entity_rule` not from this migration

Reply:
@pacospace I forgot to push changes. It should be removed now.